### ISSUE TITLE
ConfirmRegister controller and register processor are hard-coded to use modFileRegister

### ISSUE THREAD

**Issue by martinf55** [modxcms/Login#137]
Using `registry.modFileRegister` means activations fail in load balanced environments when the registration request happens on a different server to the activation request.

**Comment by Jako** [comment]
Yes, please ask someone at the LLC if they have any solutions for this.

**Comment by Jako** [comment]
Would it i.e. help to make the properties in these lines configurable by a system setting? 
https://github.com/modxcms/Login/blob/master/core/components/login/controllers/web/ConfirmRegister.php#L130-L131

Sorry, but I don't have any experience in balanced environments.

**Comment by martinf55** [comment]
Yes, I've forked the project, hopefully I can put together a pull request which allows a system setting `login.registry_class` to override the `registry.modFileRegister` with `registry.modDbRegister`. 

I've had the modDbRegister working locally, but just bumped into some kind of concurrency issue when on a load-balanced environment (possibly a lingering database lock). Once that's fixed I should have something to PR here.
